### DISCUSSION TITLE
I can't download latest kanela jar file.

### DISCUSSION THREAD

**Discussion by mkemaldurmus** [kamon-io/Kamon#1286] [Q&A] [answered]
I tried to run Kamon in IntelliJ IDEA,for fixinx some npe but I couldn't download the JAR file for fixing npe.
[Kanela releases](https://bintray.com/kamon-io/releases/kanela)

**Comment by ivantopo** [comment, marked as answer]
Try using the releases from the Maven Central repo: https://repo1.maven.org/maven2/io/kamon/kanela-agent/1.0.16/

Bintray has been gone for quite some time now!

**Comment by netanelb-carbyne** [comment]
Please fix the link [in the guide](https://kamon.io/docs/latest/guides/how-to/start-with-the-kanela-agent/#in-a-nutshell)
Also, please provide installation guides, this is far away from being cleared on how to use it
